### PR TITLE
API Docs - Section - Introduction, Authentication & others

### PR DESCRIPTION
**Pull request by atymchuk**
- [ ] No FB for this PR, API docs only changes

### CODE REVIEW

File: .github/workflows/doc.yml [hunk 1]
```diff
@@ -3,21 +3,25 @@ on:
   push:
     branches:
       - main
+  workflow_dispatch:
+
```
Comment (BupycHuk) [inline .github/workflows/doc.yml:7]: let's add inputs like api-version

Comment (BupycHuk) [reply .github/workflows/doc.yml:7]: so we will be able to create new version

File: Makefile [hunk 2]
```diff
@@ -75,6 +75,19 @@ gen: clean         ## Generate files.
...
+		api/managementpb/dbaas/json/dbaas.json \
+		api/managementpb/ia/json/ia.json \
+		api/managementpb/backup/json/backup.json \
+		api/managementpb/azure/json/azure.json \
+		api/qanpb/json/qanpb.json \
+		api/platformpb/json/platformpb.json
+	bin/swagger validate api/swagger/swagger-dev.json
+
+	bin/swagger-order --output=api/swagger/swagger-dev-only.json api/swagger/swagger-dev-only.json
+
```
Comment (BupycHuk) [inline Makefile:90]: let's discuss it tomorrow on API call

File: .github/workflows/doc.yml (hunk 1 above)
Comment (atymchuk) [reply .github/workflows/doc.yml:12]: Done!

File: Makefile [hunk 3]
```diff
@@ -75,6 +75,19 @@ gen: clean         ## Generate files.
...
+	# generate API spec with only dev PMM Server APIs specifically for readme.io (omit agentlocalpb)
+	bin/swagger mixin --output=api/swagger/swagger-dev-only.json \
+		api/swagger/header-dev.json \
+		api/managementpb/dbaas/json/dbaas.json \
+		api/managementpb/ia/json/ia.json \
+		api/managementpb/backup/json/backup.json \
+		api/managementpb/azure/json/azure.json \
+		api/qanpb/json/qanpb.json \
+		api/platformpb/json/platformpb.json
+	bin/swagger validate api/swagger/swagger-dev.json
```
Comment (BupycHuk) [inline Makefile:87]:
```suggestion
	bin/swagger validate api/swagger/swagger-dev-only.json
```

**Review by BupycHuk — APPROVED** [review]

**Review by BupycHuk — APPROVED** [review]

**Pull request merged**